spec/jobs/tasks/executor_job_spec.rb

```diff
@@ -16,4 +16,18 @@
       expect(task.processing_at).not_to eq(nil)
     end
   end
+
+  context "change the default queue name" do
```

Comment:
here there should be two context 

1. when changing the default queue name --> "it sets the queue name with changed value" 
2. when default queue name is unchanged --> "sets the queue name as default"